src/lib/main.js

```diff
@@ -165,6 +166,12 @@ module.exports = async (argv, env) => {
       error('Failed to obtain a url to the API');
       return -1;
     }
+    try {
+      await request.get(apiUrl);
```

Comment:
This will potentially hang for some time if there's a dodgy internet connection. It'd be good to output something here to let users know it may take some time, eg: `Checking that ${apiUrl} is available...`

Also do you know what the timeout condition is? How long could the user be hanging here for before the command finally fails?

Reply:
Currently using the default timeout which is the OS-wide TCP connection timeout which on linux is about 20 seconds.